src/CodeStyle/Core/CodeFixes/Host/Mef/CodeStyleHostLanguageServices.cs.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Composition.Hosting;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    internal sealed partial class CodeStyleHostLanguageServices : HostLanguageServices
+    {
+        private sealed class MefHostExportProvider : IMefHostExportProvider
+        {
+            private readonly CompositionHost _compositionContext;
+
+            public MefHostExportProvider(string languageName)
+            {
+                var assemblies = CreateAssemblies(languageName);
+                var compositionConfiguration = new ContainerConfiguration().WithAssemblies(assemblies);
+                _compositionContext = compositionConfiguration.CreateContainer();
+            }
+
+            private static ImmutableArray<Assembly> CreateAssemblies(string languageName)
+            {
+                using var disposer = ArrayBuilder<string>.GetInstance(out var assemblyNames);
+
+                assemblyNames.Add("Microsoft.CodeAnalysis.CodeStyle.Fixes");
+                switch (languageName)
+                {
+                    case LanguageNames.CSharp:
+                        assemblyNames.Add("Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes");
+                        break;
+
+                    case LanguageNames.VisualBasic:
+                        assemblyNames.Add("Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes");
+                        break;
```

Comment:
Is there ever cross-pollination between the two languages? For example, a C# code fix ends up needing to make a code change in VB code, so it ends up calling into the VB side of things?